plugins/cilium-docker/driver/ipam.go

```diff
@@ -53,6 +52,27 @@ func (driver *driver) getDefaultAddressSpaces(w http.ResponseWriter, r *http.Req
 	objectResponse(w, resp)
 }
 
+func (driver *driver) getPoolResponse(req *api.RequestPoolRequest) *api.RequestPoolResponse {
+	addr := driver.conf.Addressing
+	if req.V6 == false {
+		return &api.RequestPoolResponse{
+			PoolID: PoolIPv4,
+			Pool:   "0.0.0.0/0",
+			Data: map[string]string{
+				"com.docker.network.gateway": addr.IPV4.IP + "/32",
```

Comment:
I'm considering to include the prefix in the daemon conf instead so we don't have logic in the plugins at all.